test/functional.js

```diff
@@ -1249,22 +1075,6 @@ module.exports = {
             });
         });
 
-        it('Babel is executed on .js files', (done) => {
-            const config = createWebpackConfig('www/build', 'dev');
-            config.setPublicPath('/build');
-            config.addEntry('main', './js/arrow_function');
-
-            testSetup.runWebpack(config, (webpackAssert) => {
-                // check that babel transformed the arrow function
-                webpackAssert.assertOutputFileDoesNotContain(
-                    'main.js',
-                    '=>'
-                );
-
-                done();
-            });
-        });
-
```

Comment:
Well, it was a problem, but I'm not sure why I entirely removed it 😅

The test was failing because Babel does not apply to all the code that is generated by Webpack... and by default that one will now contain arrow functions.

I added the test back but made it check for `class` replacement instead.